docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -748,6 +750,41 @@ In this example, the table is partitioned by the month of ``order_date``, a hash
         country VARCHAR)
     WITH (partitioning = ARRAY['month(order_date)', 'bucket(account_number, 10)', 'country'])
 
+Sorted tables
+------------------
+
+In addition to partitioning, Iceberg tables may have a specified list of sort columns. When writes
+are made to the table, each file will individually be sorted by these columns. Sorting can be
+particularly helpful when selective reads are frequently made against a column with a high cardinality.
+
+Sort orders are provided using the ``sorted_by`` table property. For example::
+
+    CREATE TABLE iceberg.testdb.customer_orders (
+        order_id BIGINT,
+        order_date DATE,
+        account_number BIGINT,
+        customer VARCHAR,
+        country VARCHAR)
+    WITH (partitioning = ARRAY['country'], sorted_by = ARRAY['order_date'])
+
+Sorting can be combined with partitioning on the same column, for example if a timestamp
+column is often queried selectively but the data is sparse enough that a higher granularity
+partition transform produces a better data layout::
+
+    CREATE TABLE iceberg.testdb.customer_orders (
+        order_id BIGINT,
+        order_date DATE,
+        account_number BIGINT,
+        customer VARCHAR,
+        country VARCHAR)
+    WITH (partitioning = ARRAY['month(order_date)'], sorted_by = ARRAY['order_date'])
+
+If you would like to disable sorting for a particular operation, you can do so by setting the
+session property ``sorted_writing_enabled`` to ``false``.
+
+While the Iceberg specification supports the same set of transforms on sort columns as it does on
+partition columns, only sorting by the column's raw value is currently supported in the connector.
```

Comment:
Is it worth adding what Trino does when writing to a table with such unsupported sorting?
Does it fail, or ignore sorting?

(it should ignore)

Reply:
I feel like these two sentences are not needed or should be simplified. 

Maybe

Sorting is only performed based on the raw values of the configured columns.

Reply:
> Sorting is only performed based on the raw values of the configured columns.

That's not quite correct, at least as I read it. If you have a sort configuration of `ARRAY[truncate(a, 2), b]` we will sort the data by `b`, not `a, b`. Does that make sense?

Reply:
I have updated those sentences though, hope that's better.